app/assets/javascripts/validation.coffee

```diff
@@ -779,6 +780,54 @@ $(document).ready ->
 
             errors.push(error)
 
+  # These errors are not captured in the schema, they are business logic being
+  # enforced in the CMR. Since we cannot publish records with these conditions,
+  # we should provide that feedback to the user. Specifically, in the
+  # Additional Attribute the Value supplied by the user needs to conform to the
+  # Data Type the user selects. The checks in the errorPresent block are meant
+  # to APPROXIMATE those of CMR ingestion with the intention that there are
+  # likely some input values that the below logic will permit, and CMR will not,
+  # but never vice versa.
+  validateAdditionalAttributeValueField = (errors) ->
+    # pulling the regex's from the hidden field tags used to share them between
```

Comment:
I don't think this is a big deal, so not worth a lot of time. But I'm wondering if it's worth trying a guard clause to exit out of this method if we're not on the right form and avoid setting up the regex vars for no reason. Just a thought - probably it doesn't save a lot of time

Reply:
Ohh good point. I think it's worth doing